hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioPipeline.java

```diff
@@ -263,6 +253,7 @@ public void onError(Throwable error) {
             currentThread().interrupt();
         }
 
+        SelectionKey selectionKey = this.selectionKey;
```

Comment:
Null protection. The selectionKey can be null, so this prevents running into a NPE when calling this method.